HLTriggerOffline/Higgs/python/hltHiggsValidator_cfi.py

```diff
@@ -265,7 +265,7 @@
             "HLT_L1_TripleJet_VBF_v"
             ),
         recJetLabel  = cms.string("ak4PFJetsCHS"),
-        jetTagLabel  = cms.string("pfCombinedSecondaryVertexBJetTags"),
```

Comment:
"CombinedSecondaryVertex" without "V2" is really a legacy version of CSV but run in the candidate-based framework. I was therefore surprised to see it being monitored here particularly because it is not one of the supported taggers. Maybe @silviodonato can comment?